e2e_playwright/st_vega_lite_chart_test.py

```diff
@@ -102,9 +105,21 @@ def test_vega_lite_chart_user_supplied_colors(
 ):
     """Tests that st.vega_lite_chart respects user configuration"""
     vega_lite_charts = themed_app.get_by_test_id("stArrowVegaLiteChart")
-    expect(vega_lite_charts).to_have_count(13)
+    # expect statement here so that snapshots are taken properly
+    expect(vega_lite_charts).to_have_count(14)
```

Comment:
but what you can do to simplify adding new test cases is to store the number of charts into a top level constant